plugins/xsts/hu.bme.mit.gamma.statechart.lowlevel.transformation/src/hu/bme/mit/gamma/statechart/lowlevel/transformation/ActionTransformer.xtend

```diff
@@ -77,6 +83,37 @@ class ActionTransformer {
 		return result.wrap
 	}
 	
+	protected def transformDoActions(Collection<? extends Action> actions, State state) {
+		for (action : actions) {
+			action.transformDoAction(state)
+		}
+	}
+	
+	protected dispatch def transformDoAction(Action action, State state) {
+
+	}
+	
+	protected dispatch def transformDoAction(CallActivityAction action, State lowlevelState) {
```

Comment:
Use the same parameter names if they refer to the same thing in dispatch methods: `state` vs. `lowlevelState`